pycloudlib/kvm/cloud.py

```diff
@@ -49,48 +50,88 @@ def get_instance(self, instance_name):
         """
         return KVMInstance(instance_name)
 
-    def launch(self, name, release, inst_type=None, wait=True):
+    def launch(self, image_id, instance_type=None, user_data=None,
+               wait=True, name=None):
         """Set up and launch a container.
 
         This will init and start a container with the provided settings.
         If no remote is specified pycloudlib defaults to daily images.
 
         Args:
-            name: string, what to call the instance
-            release: string, [<remote>:]image, what release to launch
-            inst_type: string, type to use
+            image_id: string, [<remote>:]image, what release to launch
```

Comment:
maybe worth mentioning that <remote> is either 'release' or 'daily' for multiplass/kvm